src/load-config.js

```diff
@@ -91,6 +91,23 @@ const finalConfig = {
           ...(config.https || {}),
         }
       : config.https,
+  upload:
+    process.env.ACTUAL_FILE_SYNC_LIMIT ||
+    process.env.ACTUAL_SYNC_ENCRYPTED_FILE_LIMIT ||
+    process.env.ACTUAL_FILE_LIMIT
+      ? {
+          fileSyncLimit:
+            +process.env.ACTUAL_FILE_SYNC_LIMIT ||
+            +process.env.ACTUAL_FILE_SIZE_LIMIT_MB ||
+            20,
+          syncEncryptedFileLimit:
+            +process.env.ACTUAL_SYNC_ENCRYPTED_FILE_LIMIT ||
+            +process.env.ACTUAL_FILE_SIZE_LIMIT_MB ||
+            50,
+          fileLimit: +process.env.ACTUAL_FILE_LIMIT || 20,
+          ...(config.upload || {}),
+        }
+      : config.upload,
```

Comment:
Actually, I think there is one more bug.

If the use has not provided these env vars and he is not using a config file: the `fileSizeLimitMB` vars will not be set. Which means `app.js` will break.

Reply:
I've added a section to `defaultConfig` for `upload`. That should resolve this